ddtrace/tracer/remote_config.go

```diff
@@ -150,6 +151,19 @@ func (t *tracer) onRemoteConfigUpdate(u remoteconfig.ProductUpdate) map[string]s
 		if updated {
 			telemConfigs = append(telemConfigs, t.config.globalTags.toTelemetry())
 		}
+		if c.LibConfig.Enabled != nil {
+			if t.config.enabled.current == true && *c.LibConfig.Enabled == false {
+				log.Debug("APM tracing disabled via remote config. Restart the service to enable it.")
+				updated = t.config.enabled.handleRC(c.LibConfig.Enabled)
+				if updated {
+					telemConfigs = append(telemConfigs, t.config.enabled.toTelemetry())
+				}
+			}
+			if t.config.enabled.current == false && *c.LibConfig.Enabled == true {
+				log.Debug("Disabling tracing via remote config. No traces will be sent.")
+				telemConfigs = append(telemConfigs, t.config.enabled.toTelemetry())
+			}
```

Comment:
I'm confused by this one...
The current value is `false` and someone tries to set it to `true`, which isn't allowed without restart, but then it also says "Disabling tracing via remote config. No traces will be sent."?

If I'm interpreting this right, we would reach this code if someone was trying to re-enable tracing without a proper restart, at which point we should consider logging a different warning/error indicating that re-enabling APM tracing requires a restart, and presumably not change telemetry.

Reply:
You are right, i willl change the log to indicate that no change will be applied and a restart is needed. 
I will remove the telemetry too, as it's supposed to indicate change only, it was a point of discussion with other folks during last sync, this one slipped through

Reply:
Okay sounds good.
(In that case, I also suggest changing it from an if to an else-if)